client/src/plugins/zeebe-plugin/deployment-plugin/DeploymentPluginValidator.js

```diff
@@ -311,3 +313,52 @@ function shallowEquals(a, b) {
 function validCloudUrl(url) {
   return /^(https:\/\/|)[a-z\d-]+\.[a-z]+-\d+\.zeebe\.camunda\.io(:443|)\/?/.test(url);
 }
+
+const checkNumberLimits = (string, min, max) => {
+  const number = parseInt(string);
+  const ret = !isNaN(number) && number >= min && number <= max;
+  return ret;
+};
+
+const validateUrl = (value, message) => {
+  let fullURL = value;
+
+  if (!(value.startsWith('https://') || value.startsWith('http://'))) {
+    fullURL = 'http://' + value;
+  }
+
+  try {
+    new URL(fullURL);
```

Comment:
Note that we can and I think also should use the parsed URL to validate the port. No need to do custom regex magic here.

Reply:
It does, but it doesn't make it mandatory. Here we want to make it clear that the port is needed.

I also realised, by adding `https://` when it's missing, this also validates the IP address.

Reply:
So I changed the port validation function to check that the port is there, and let `new URL()` handle the rest.